test/plots/scatterPlotTests.ts

```diff
@@ -19,11 +19,17 @@ describe("Plots", () => {
             .y((d: any) => d.y, yScale);
       });
 
+      afterEach(function() {
+        if (this.currentTest.state === "passed") {
```

Comment:
this this kind of test reflection common? is there a harm in always destroying the plot and removing the svg?